### PR TITLE
Update tested AGP versions

### PR DESCRIPTION
**Pull request by eskatos**
* from 8.0.0-alpha11 to 8.0.0-beta02
* introducing 8.1.0-alpha03

This is the first update that include AGP versions that require Gradle 8.0

### CODE REVIEW

**Comment by eskatos** [comment]
@bot-gradle test this

**Comment by eskatos** [comment]
@bot-gradle test APT

**Review by jbartok — APPROVED** [review]

**Comment by eskatos** [comment]
@bot-gradle test and merge

File: subprojects/docs/src/docs/userguide/compatibility.adoc [hunk 1]
```diff
@@ -79,5 +79,5 @@ Gradle is tested with Groovy 1.5.8 through 4.0.0.
 Gradle plugins written in Groovy must use Groovy 3.x for compatibility with Gradle and Groovy DSL build scripts.
 
 == Android
-Gradle is tested with Android Gradle Plugin 7.3 and 7.4.
+Gradle is tested with Android Gradle Plugin 7.3, 7.4 and 8.0.
```
Comment (ljacomet) [inline subprojects/docs/src/docs/userguide/compatibility.adoc:82]: And 8.1? Or do we announce that only after `alpha`?

**Review by ljacomet — APPROVED** [review]
LGTM!

Minor comment on the docs change

File: subprojects/docs/src/docs/userguide/compatibility.adoc (hunk 1 above)
Comment (eskatos) [reply subprojects/docs/src/docs/userguide/compatibility.adoc:82]: We usually only explicitly list >= beta.

**Comment by eskatos** [comment]
@bot-gradle test and merge

**Pull request merged**